docs/site/todo-tutorial-geocoding-service.md

```diff
@@ -10,12 +10,12 @@ summary: LoopBack 4 Todo Application Tutorial - Integrate with a geo-coding serv
 
 ### Services
 
-To call
-[other APIs and web services](./Calling-other-APIs-and-web-services.html) from
-LoopBack applications, we recommend to use Service Proxies as a design pattern
-for encapsulating low-level implementation details of communication with
-3rd-party services and providing JavaScript/TypeScript API that's easy to
-consume e.g. from Controllers.
+To call other APIs and web services from LoopBack applications, we recommend to
+use Service Proxies as a design pattern for encapsulating low-level
+implementation details of communication with 3rd-party services and providing
+JavaScript/TypeScript API that's easy to consume e.g. from Controllers. See
+[Calling other APIs and web services](./Calling-other-APIs-and-web-services.html)
```

Comment:
`.md` here so that users browsing the docs from github can go to a valid link

Reply:
Good catch, fixed.